USAGE.adoc

```diff
@@ -0,0 +1,52 @@
+= How to use this tool
```

Comment:
Can you also change the README.md to indicate the purpose of this tool. IIUC it has changed with your improvements, now it does more than find deprecated API. Actually, it should deserve a change on its name, but as it's a tool I don't think is worth it.

Reply:
Facelift applied to the README. 

Likely will convert it to asciidoc later, but that's OK for now.